spoc/pixels.py

```diff
@@ -198,43 +201,45 @@ class GenomicBinner:
 
     Args:
         bin_size (int): The size of the genomic bins.
-
     """
 
     def __init__(self, bin_size: int) -> None:
         self._bin_size = bin_size
-        self._contact_order = None
 
-    def _get_assigned_bin_output_structure(self):
-        columns = [f"chrom_{index}" for index in range(1, self._contact_order + 1)] + [
-            f"start_{index}" for index in range(1, self._contact_order + 1)
+    def _get_assigned_bin_output_structure(self, contact_order: int):
```

Comment:
yes, this was to solve a mypy problem that `self._contact_order` could be `None` and that if we pass it to this method the type checker fails, because it's not smart enough to realize that everywhere we use this method, it will be not None. In any case, i think it's cleaner to explicitely pass the contact_order